src/ducks/_Table/index.js

```diff
@@ -39,40 +38,50 @@ const Table = ({
   itemProps,
   isLoading,
   getItemKey,
-  onRowClick
+  onRowClick,
+  isWithColumnTitles = true,
+  emptySection,
+  ...props
 }) => (
   <table className={cx(styles.wrapper, className)}>
-    <thead>
-      <tr>
-        {columns.map(({ id, title, Title }) => (
-          <th key={id}>{Title ? <Title {...itemProps} /> : title}</th>
-        ))}
-      </tr>
-    </thead>
+    {isWithColumnTitles && (
+      <thead>
+        <tr>
+          {columns.map(({ id, title, Title }) => (
+            <th key={id}>{Title ? <Title {...itemProps} /> : title}</th>
+          ))}
+        </tr>
+      </thead>
+    )}
     <tbody>
-      {items.map((item, i) => {
-        const itemIndex = offset + i
-
-        return (
-          <tr
-            key={getItemKey ? getItemKey(item) : item[itemKeyProperty]}
-            onClick={onRowClick && (e => onRowClick(item, e))}
-          >
-            {columns.map(({ id, render, className }) => (
-              <td key={id} className={className}>
-                {render(item, itemProps, itemIndex)}
-              </td>
-            ))}
-          </tr>
-        )
-      })}
-      {minRowsPadding(minRows, columns, items)}
+      {!isLoading && items.length === 0 && emptySection && (
+        <tr className={styles.disableHover}>
+          <td colSpan={columns.length}>{emptySection}</td>
+        </tr>
+      )}
+      {items.length > 0 && (
+        <>
+          {items.map((item, i) => {
+            const itemIndex = offset + i
+            return (
+              <tr
+                key={getItemKey ? getItemKey(item) : item[itemKeyProperty]}
+                onClick={onRowClick && (e => onRowClick(item, e))}
+              >
+                {columns.map(({ id, render, className }) => (
+                  <td key={id} className={className}>
+                    {render(item, itemProps, itemIndex)}
+                  </td>
+                ))}
+              </tr>
+            )
+          })}
+          {minRowsPadding(minRows, columns, items)}
+        </>
+      )}
```

Comment:
`items.length > 0 &&` breaks the `minRowsPadding` behaviour in case of empty array. You can set `minRows` to `0` in the component that uses `Table` if it's needed